rest-villains/pom.xml

```diff
@@ -4,7 +4,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>io.quarkus.sample.super-heroes</groupId>
   <artifactId>rest-villains</artifactId>
-  <version>1.0</version>
+  <version>java17-latest</version>
```

Comment:
Maybe we can simply copy back the `pom.xml.versionsBackup` into the `pom.xml`

Reply:
We don't need to do anything as the script that commits to version control doesn't  commit pom.xml. The commit of this file in this change set was a mistake.

Reply:
Got it. 
I usually prefer not changing things I don't need to change, but I see that the github workflow step does only add changes in the deploy folder, so I'm ok not reverting the changes during the workflow.
Anyway, yes, we need to undo this change here.

Reply:
Not modifying pom.xml isn't really an option because of the way we're using `./mvnw versions:set` to set the project version in the scripts. We do the same thing in the `generate-k8s-resources.sh` script.

Reply:
No, that's now what I meant. I understand we need to use`versions:set` to update the `pom.xml` before generating the charts.
My proposal was to run `versions:revert` afterwards. Something like `  $project/mvnw -q -f $project/pom.xml versions:set clean package versions:revert \```.

Reply:
Got it.

I mean we _could_ do that as part of our loop, but at the end of the day we are not committing the file into version control, so given that we are doing a matrix of 2 JVM versions x 4 projects x 2 "kinds" (JVM vs native) x 4 deployment types (kubernetes/minikube/openshift/knative) I think it would just be wasted CPU cycles that would take extra time to a job that already takes an hour or so to run.

Just my opinion.